[TASK] Add new user for users with do not have id 1000 (#28)
pluswerk/php-dev: README.md

#      - PIMCORE_ENVIRONMENT=development_docker
#      - TYPO3_CONTEXT=Development/docker

>       # If your current user is not uid 1000, get in in container via: docker-compose exec -u ${UID} web bash

Reviewer: ```suggestion
      # If your current user is not uid 1000 set it via .env file. Don't forget to connect via ./start.sh
```

>       - APPLICATION_UID=1001

Reviewer: ```suggestion
      - APPLICATION_UID=${APPLICATION_UID:-1000}
```

>       - APPLICATION_GID=1001

Reviewer: ```suggestion
      - APPLICATION_GID=${APPLICATION_GID:-1000}
```
